x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/use_columns.tsx

```diff
@@ -150,7 +151,26 @@ export const useColumns = (
       ),
       width: '30px',
       render: (item) => {
-        return Array.isArray(item.alerting_rules) ? (
+        const needsReauth = needsReauthorization(item);
+
+        const needsReauthTooltipIcon = needsReauth ? (
+          <>
+            <EuiToolTip
+              content={i18n.translate(
+                'xpack.transform.transformList.needsReauthorizationBadgeTooltip',
+                {
+                  defaultMessage:
+                    'This transform was created with insufficient permissions. You must have the transform_admin built-in role or manage_transform cluster privileges to reauthorize and start operations.',
```

Comment:
Likewise since transform_admin is just another option for getting manage_transform privileges, IMO it can be omitted:
```suggestion
                    'This transform was created with insufficient permissions. You must have manage_transform cluster privileges to reauthorize and start it.',
```

Reply:
Updated here [`3f64b1f` (#154736)](https://github.com/elastic/kibana/pull/154736/commits/3f64b1f53667da00f811cc2579eb31decf81fafb)